modules/social_features/social_group/social_group.module

```diff
@@ -229,10 +229,10 @@ function social_group_preprocess_group(array &$variables) {
   $variables['group_type_id'] = $group_type_id;
   $variables['group_type'] = $group->getGroupType()->label();
 
-  // From 10.0.0 onwards, we want to show group types differently.
```

Comment:
@rollki @ribel Any intention for removal of this comment. We introduced a new "Group type" feature in 10.0.0 here: https://github.com/goalgorilla/open_social/pull/2140/files